tests/unit/Core/Content/Sitemap/Provider/CategoryUrlProviderTest.php

```diff
@@ -89,8 +121,141 @@ public function testGetCategoryUrls(): void
         static::assertSame('category/2/detail', $url->getLoc());
     }
 
-    private function initServices(): void
+    public function testEmptyUrlResult(): void
```

Comment:
The naming could be improved, you do not write which method is tested, this makes harder to understand the idea behind it. Try testGetUrlsReturnsNothing or testGetUrlsReturnsEmptyUrlResult or something.